source/read/change-streams.txt

```diff
@@ -0,0 +1,284 @@
+
+.. literalinclude:: /includes/read/change-streams.kt
+   :start-after: start-update-for-change-stream
+   :end-before: end-update-for-change-stream
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+When you update the collection, the change stream application prints the change
+as it occurs. The printed change event resembles the
+following:
+
+.. code-block:: json
+   :emphasize-lines: 14-20
+
+   {
+     "cursor": {
+       "nextBatch": [
+         {
+           "_id": { ... },
+           "operationType": "update",
+           "clusterTime": { ... },
+           ...
+           "ns": {
+             "db": "sample_restaurants",
+             "coll": "restaurants"
+           },
+           ...
+           "updateDescription": {
+             "updatedFields": {
+               "cuisine": "Irish"
+             },
+             "removedFields": [],
+             "truncatedArrays": []
+           }
+         }
+       ],
+       ...
+     }
+   }
+
+Modify the Change Stream Output
+-------------------------------
+
+You can pass the ``pipeline`` parameter to the ``watch()`` method to modify the
+change stream output. This parameter allows you to watch for only specified
+change events. Format the parameter as a list of objects that each represents an
+aggregation stage.
+
+You can specify the following stages in the ``pipeline`` parameter:
+
+- ``$addFields``
+- ``$match``
+- ``$project``
+- ``$replaceRoot``
+- ``$replaceWith``
+- ``$redact``
+- ``$set``
+- ``$unset``
+
+Match Specific events Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example uses the ``pipeline`` parameter that includes a ``$match``
+to open a change stream that only records update operations:
+
+.. literalinclude:: /includes/read/change-streams.kt
+   :start-after: start-change-stream-pipeline
+   :end-before: end-change-stream-pipeline
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+To learn more about modifying your change stream output, see the
+:manual:`Modify Change Stream Output
+</changeStreams/#modify-change-stream-output>` section in the {+mdb-server+}
+manual.
+
+Modify watch() Behavior
+-----------------------
+
+You can modify the ``watch()`` by chaining methods to the ``ChangeStreamIterable``
+object returned by the ``watch()`` method call. If you don't specify any options, the
+driver does not customize the operation.
+
+The following table describes methods you can use to customize the behavior
+of ``watch()``:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``batchSize()``
+     - | Sets the number of documents to return per batch.
+
+   * - ``collation()``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``comment()``
+     - | Specifies a comment to attach to the operation.
+
+   * - ``forEach()``
```

Comment:
This is not really customization.  Rather, it's one of several methods that can be used to iterator the change stream cursor.  It can be removed from the table.

Reply:
Will remove in next revision